### DISCUSSION TITLE
Plausible as available Service

### DISCUSSION THREAD

**Discussion by rafa-acioly** [vitodeploy/vito#406] [Ideas]
Plausible is intuitive, lightweight and open source web analytics. No cookies and fully compliant with GDPR, CCPA and PECR.

TLDR;
A google analytics open source alternative.

Official website: https://plausible.io/open-source-website-analytics
Repository: https://github.com/plausible/analytics

**Comment by saeedvaziry** [comment]
I suggest developing a plugin for this

https://vitodeploy.com/docs/plugins
